tools/aa/commands/AddRepoCommand.java

```diff
@@ -75,7 +75,7 @@ public boolean run(String[] args) {
     String repoPath = fileUtils.joinPaths(headPath, repoName);
     System.out.println(String.format("Cloning repo %s into %s", repoName, repoPath));
     GitRepo repo = this.gitRepoFactory.create(repoPath);
-    if (!repo.cloneRepo(url.get(), repoPath)) {
+    if (!repo.cloneRepo(url.get(), fileUtils.joinPaths(repoPath, ".git"))) {
```

Comment:
Let's add this to `cloneRepo`, so people don't have to remember to add the `.git` every time.